build-support/scripts/protobuf.sh

```diff
@@ -79,6 +84,28 @@ function main {
     return 0
 }
 
+# List all of the files in the current buf module and check for a valid version string.
+# A version string is treated as the second element in a directory path. E.g. /path/[VERSION]/file.proto
+function validate_resource_version {
+	for FILE in $(buf ls-files)
```

Comment:
When I run `buf ls-files` I don't get the full path which it looks like you're expecting to parse.

```
$ buf ls-files                                                                                                                                                                                                                              
annotations.proto
resource.proto
```